src/packages/calendar/calendarviewmodeitem.taro.tsx

```
@@ -0,0 +1,843 @@
+                defaultArr.push(item)
+                obj[item] = item
+              }
+            }
+          })
+          currentDate.splice(0, currentDate.length, ...defaultArr)
+          defaultData = [...splitDate(defaultArr[0])]
+          break
+        case 'week':
+          // eslint-disable-next-line no-case-declarations
+          const [y, m, d] = splitDate(currentDate[0])
+          // eslint-disable-next-line no-case-declarations
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)